app/models/manageiq/providers/vmware/inventory/parser/cloud_manager.rb

```diff
@@ -0,0 +1,119 @@
+class ManageIQ::Providers::Vmware::Inventory::Parser::CloudManager < ManageIQ::Providers::Vmware::Inventory::Parser
+  def parse
+    orgs
+    vdcs
+    vapps
+    vms
+    vapp_templates
+    images
+  end
+
+  private
+
+  def orgs
+    collector.orgs
+  end
```

Comment:
Remove this function ^. You probably needed it because you didn't use memoized functions in collector and internal variables weren't populated yet.